### PR TITLE
Added mention of OpenAPI support for gRPC JSON Transcoding

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal Review - gRPC Transcoding section](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-7.0?view=aspnetcore-7.0&branch=pr-en-us-27331#json-transcoding)
Contributes to #26477 

Added mention of OpenAPI support as sub item to gRPC JSON Transcoding section.

### CODE REVIEW

**Comment by wadepickett** [comment]
@JamesNK, I neglected to mention OpenAPI support for gRPC JSON Transcoding.   I added mention of it here: [Internal Review - gRPC Transcoding section](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-7.0?view=aspnetcore-7.0&branch=pr-en-us-27331#json-transcoding).   Do you want it in its own section or does this look OK how it is?

File: aspnetcore/release-notes/aspnetcore-7.0.md [hunk 1]
```diff
@@ -192,8 +192,9 @@ gRPC JSON transcoding is an extension for ASP.NET Core that creates RESTful JSON
 
 * Apps to call gRPC services with familiar HTTP concepts.
 * ASP.NET Core gRPC apps to support both gRPC and RESTful JSON APIs without replicating functionality.
+* Support for generating OpenAPI from transcoded RESTful APIs by integrating with with [Swashbuckle](xref:tutorials/get-started-with-swashbuckle).
```
Comment (JamesNK) [inline aspnetcore/release-notes/aspnetcore-7.0.md:195]:
```suggestion
* Experimental support for generating OpenAPI from transcoded RESTful APIs by integrating with with [Swashbuckle](xref:tutorials/get-started-with-swashbuckle).
```

**Comment by wadepickett** [comment]
@JamesNK, sorry, I see I had left "draft" on as you provided the review.  I will assume it is approved with your commit in.  Thanks.

**Comment by JamesNK** [comment]
Opps, I merged instead of approved. I approve 😄

**Pull request merged**